unblob/cli.py

```diff
@@ -39,7 +39,10 @@ def show_external_dependencies(
     extra_handlers = plugin_manager.load_handlers_from_plugins()
     handlers = ctx.params["handlers"] + tuple(extra_handlers)
 
-    dependencies = get_dependencies(handlers)
+    extra_dir_handlers = plugin_manager.load_dir_handlers_from_plugins()
+    dir_handlers = ctx.params["dir_handlers"] + tuple(extra_dir_handlers)
+
+    dependencies = get_dependencies(handlers, dir_handlers)
```

Comment:
I see a lot of code being duplicated to support `handlers` and `dir_handlers` separately. Maybe having a generalized `Handler` class would simplify things ? With a hierarchy like this:

```
Handler
| -> ChunkHandler
| -> DirectoryHandler
StructHandler
```

Which would also mean that if I want to do some C parsing within my directory handler I just need to inherit from both `DirectoryHandler` and `StructHandler`.

Reply:
Even outside of this MR, turning `StructHandler` into a mixin sounds like a good idea btw.

Reply:
I would rather create a StructHandlerMixIn and keep the current SturctHandler use it.

In the meantime we can use StructParser directly as well. Also we should look at the current handlers to see what is the typical pattern of using a StructParser or using a format parser which also handler extraction.